sdk/videoanalyzer/Azure.Media.VideoAnalyzer.Edge/README.md

```diff
@@ -17,17 +17,17 @@ This is a models-only SDK. All client operations are done using the [Microsoft A
 
 The client is coming from Azure IoT SDK. You will need to obtain an [IoT device connection string][iot_device_connection_string] in order to authenticate the Azure IoT SDK. For more information please visit: [https://github.com/Azure/azure-iot-sdk-csharp].
 
-```C# Snippet:Azure_VideoAnalyzer_Samples_ConnectionString
-var connectionString = "connectionString";
-this._serviceClient = ServiceClient.CreateFromConnectionString(connectionString);
+```C# Snippet:Azure_VideoAnalyzerSamples_ConnectionString
+String connectionString = "HostName=lvasamplehubnpctns45jvoji.azure-devices.net;SharedAccessKeyName=iothubowner;SharedAccessKey=dburWgVNqX+/xuOpSfXMKQdkr4nypY7HbwmwduL3mbs=";
```

Comment:
You need to actually change the sample source and rerun `eng/scripts/Update-Snippets.ps1 -ServiceDirectory videoanalyzer`. Builds won't pass otherwise since they also run these scripts and compare.

I'll sign off, but you can't merge until this is done.